backend/control_center/ansible/playbooks/resources/ovs-flow-check/check_ovs_flows.sh

```
@@ -0,0 +1,78 @@
+# Flow 3: Send to Controller (miss rule)
+FLOW_CONTROLLER_COOKIE="0xa"
+FLOW_CONTROLLER_RULE="cookie=${FLOW_CONTROLLER_COOKIE},priority=0,actions=CONTROLLER:65535"
+
+# --- Check and Add Logic ---
+# Use a single `dump-flows` call for efficiency
+CURRENT_FLOWS=$(sudo ovs-ofctl dump-flows ${BRIDGE} -OOpenflow13 2>> "${LOG_FILE}")
+
+# Check for ARP flow
+if ! echo "${CURRENT_FLOWS}" | grep -q "cookie=${FLOW_ARP_COOKIE}"; then
+    log_message "ARP flow (cookie ${FLOW_ARP_COOKIE}) is MISSING. Re-adding..."
+    sudo ovs-ofctl add-flow ${BRIDGE} "${FLOW_ARP_RULE}" -OOpenflow13 >> "${LOG_FILE}" 2>&1
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use .. | sudo tee -a file

(SC2024)